mautic/mautic
# FormJS Fix for issue #3344

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #3344 
| BC breaks? | n/a
| Deprecations? | n/a

#### Description:
This PR fixes an issue with `mautic-form.js`. Currently the script does not complete initialization correctly because the name of the source file is different in production vs dev (`mautic-form-src.js`).

#### Steps to test this PR:
1. Install this PR and reload a form that is embeded in a remote site.
2.  See issue #3344 for more details on verifying if the script is loaded correctly
3. Run `php app\console mautic:assets:generate` and verify that the freshly generated `mautic-form.js` also has the correct script name.

#### Steps to reproduce the bug:
See issue #3344 for more details
